lib/primitives/txmeta.js

```diff
@@ -90,7 +90,10 @@ class TXMeta {
   /**
    * Inject properties from options object.
    * @private
-   * @param {Object} options
+   * @param {TX} tx
+   * @param {ChainEntry} entry
+   * @param {Number} index
```

Comment:
funny I dont see `entry` or `index` actually used anywhere this function is called. I doubt it was intended to be a `ChainEntry` anyway, more likely a `MempoolEntry` but I'm having a hard time confirming

Reply:
yeah, that sounds right!!